lib/barcodes_panel.dart

```diff
@@ -80,16 +115,20 @@ class BarcodesPanelState extends State<BarcodesPanel> {
       child: Column(
         children: [
           Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
             children: [
-              _textField(_descriptionController, "Opis"),
+              _textField(_descriptionController, "Opis", _descriptionError),
               const SizedBox(width: 10),
-              _textField(_dataController, "Kod kreskowy"),
+              _textField(_dataController, "Kod kreskowy", _dataError),
               const SizedBox(width: 10),
               _dropdownField(),
               const SizedBox(width: 10),
-              ElevatedButton(
-                onPressed: _handleAddBarcode,
-                child: const Text('Dodaj kod kreskowy'),
+              Padding(
+                padding: const EdgeInsets.only(top: 20.0), // Wyrównanie przycisku z polami tekstowymi
```

Comment:
Często pojawia się tu 20 i 10, które opisują te same wartości. Stwórz na nie prywatną klasę Constants i zamiast wartości wpisanej ręcznie daj np. `Constants.errorHeight`

Reply:
To samo z 10 która jest wyżej.